charmander/config.py

```diff
@@ -0,0 +1,33 @@
+"""
+Configure the application.
+
+"""
+from microcosm.config.model import Configuration
+
+
+def load_default_config(metadata):
+    """
+    Construct application default configuration.
+
+    There should be very little here.
+
+    """
+    config = Configuration(
```

Comment:
Here is where you configure the defaults for the app. Most of this should be left alone, the port for local service configuration should be set to something safe for your environment.